decidim-core/app/views/layouts/decidim/mailer.html.erb

```diff
@@ -123,6 +123,19 @@
                 </tr>
               </table>
             <% end %>
+            <table class="main container">
```

Comment:
I would like to propose a couple of changes to the styling of the extra footer:
1. Can we make the background the same as the surrounding area around the email
2. Can we add a slight 10px margin/padding on top of the element to separate it a bit from the actual email body

The end result should look something like this after these changes:
![Email footer changes](https://user-images.githubusercontent.com/864340/184080766-69ff9e39-1630-4f93-9c70-4eda282736a3.png)